test/fixtures/core.scale/tick-backdrop.js

```diff
@@ -71,6 +71,6 @@ module.exports = {
       height: 256,
       width: 256
     },
-    spriteText: true,
+    spriteText: false,
```

Comment:
This is likely the cause of test failures. We should always `spriteText: true` to account for browser differences

Reply:
spriteText overwrites measureText() which is needed both for my existing code and also for the textdecoration functionality - I don't know how that's passing tests with spriteText set to true (if it has coverage). I can look into a PR into the test-utils to change how spriteText works. I'm guessing that it was implemtned to overwrite measureText() before it was fully supported, and now that it's got wide browser support we can probably ditch the measureText() overwrite from test-utils.

Reply:
Also should note that when running tests locally on MacOS Chrome/Firefox I didn't get any failures on my new tests, though I did get one unrelated failure.

Reply:
Fonts are rendered differently based on os/browser combination. Thats why we have the custom (ugly) font drawing in our test utils, to have something font like that works on all platforms. Previously we just avoided any text in fixtures.

Reply:
@kurkle , that makes sense for fonts - but what spriteText also disables is the full TextMetrics implementation of meatureText(), e.g. [actualBoundingBoxLeft](https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics/actualBoundingBoxLeft). As you can see from browser-support, it doesn't appear that spriteText needs to continue to override measureText.

Reply:
measureText is supported alright, but browsers give different results for the same font (and thus render a little bit differently). Also spriteText is not using the actual font specified, so measureText can't give correct results.

It would probably be good idea to add a crude implementation of the TextMetrics to chartjs-test-utils. (colors would also be nice)